Formula/fff.rb

```diff
@@ -0,0 +1,18 @@
+class Fff < Formula
+  desc ":rocket: fucking fast file-manager"
+  homepage "https://github.com/dylanaraps/fff"
+  url "https://github.com/dylanaraps/fff/archive/1.0.tar.gz"
+  sha256 "6e5eed18b50b3e1a2497092e618665ff00b71b27b4f7653f9693ee40517d1a8a"
+
+  depends_on "bash"
+  depends_on "coreutils"
+
+  def install
+    bin.install "fff"
+    man1.install "fff.1"
+  end
+
+  test do
+    system bin/"fff", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
It's fully interactive. The only non-interactive aspect is `fff -v`.

Reply:
How should I test the interactive script?

Reply:
Do I need to test the interactivity of the script? If so, is there a specific way you'd prefer I test it? I don't exactly know what resources I have access to in the testing enviroment

Reply:
Marking as resolved as no suggestions on how to properly test an interactive script was provided.

I assume this shall be fine for now as the [https://docs.brew.sh/Formula-Cookbook](Formula Cookbook) says the following:

> We want tests that don’t require any user input and test the basic functionality of the application. For example foo build-foo input.foo is a good test and (despite their widespread use) foo --version and foo --help are bad tests. **However, a bad test is better than no test at all.**

I am willing to fix this if needs be, but I need input as to how you would prefer it to be done